lib/util/SortableSet.js

```diff
@@ -94,4 +92,16 @@ module.exports = class SortableSet extends Set {
 		this._cacheOrderIndependent.set(fn, newData);
 		return newData;
 	}
-};
+
+	_invalidateCache(all) {
+		if(this._cache !== null)
```

Comment:
`if(this._cache !== undefined)`

When always using `undefined` instead of `null` for setting and comparing `this._cache` and `this._cacheOrderIndependent` you can get rid of properties on the prototype.